smoothie.js

```diff
@@ -1072,7 +1072,7 @@
         var gy = dimensions.height - Math.round(v * stepPixels);
         var yValue = chartOptions.yIntermediateFormatter(this.valueRange.min + (v * step), labelsOptions.precision);
         //left of right axis?
-        intermediateLabelPos =
+        let intermediateLabelPos =
```

Comment:
I think it's better to use `var` instead of `let` because older browsers don't support it and because `var` is never used anywhere else in this library.

Reply:
Oh, and also perhaps it would be better to replace the `;` with `,` in the line above, where another variable is declared, and add indentation, so two variables are declared with a single `var` statement, like it's done accross the codebase.

Reply:
I see this is the same syntax used on the lines above; changed in the new commit!